runtimes/cpp-17/.env

```diff
@@ -1,2 +1,4 @@
-INTERNAL_RUNTIME_KEY=secret-key
-INTERNAL_RUNTIME_ENTRYPOINT=index.cc
\ No newline at end of file
+OPEN_RUNTIMES_SECRET=secret-key
+OPEN_RUNTIMES_ENTRYPOINT=index.cc
+BUILD_COMMAND=""
```

Comment:
Do we want an `OPEN_RUNTIMES` prefix for both `BUILD_COMMAND` and `START_COMMAND`?

Reply:
Good catch, it started as just test variable 😅 Updated.